src/ua/burkavtsov/HW24/JBDC.java

```diff
@@ -0,0 +1,62 @@
+package ua.burkavtsov.HW24;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class JBDC {
+    private final Connection connection;
+
+    public JBDC(Connection connection) {
+        this.connection = connection;
+    }
+
+    public void create(Entity entity) throws SQLException {
+        String sql = "INSERT INTO entity_table (column1, column2) VALUES (?, ?)";
+        try (PreparedStatement statement = connection.prepareStatement(sql)) {
+            statement.setString(1, entity.getProperty1());
+            statement.setString(2, entity.getProperty2());
+
+            statement.executeUpdate();
+        }
+    }
+
+    public ReaderEntity(int entityId) throws SQLException {
```

Comment:
N: method name should be in lowercase (read or readEntity)

Also, entity by some id can be not present in DB, so you can/have use Optional API. If entity exists, return Optional.of(entity), if not, return Optional.empty();

```public Optional<Entity> read(int id)```